### PR TITLE
Apple Pay in Sample App

### PR DESCRIPTION
**Pull request by dbart01**
### What this does

Adds a mocked  Pay implementation in the sample app. There are more PRs to come that build and modify some of these futures. They're coming in fast.

### CODE REVIEW

File: Sample Apps/Storefront/Storefront/CartViewController.swift [hunk 1]
```diff
@@ -25,13 +25,16 @@
...
 import UIKit
+import Pay
 
 class CartViewController: ParallaxViewController {
 
     @IBOutlet fileprivate weak var tableView: UITableView!
     
     private var totalsViewController: TotalsViewController!
     
+    fileprivate let payController = PayController(merchantID: "greats-clone.myshopify.com")
```
Comment (davidmuzi) [inline Sample Apps/Storefront/Storefront/CartViewController.swift]: ideally these constants are pulled from an environment variable, or some struct with static properties

File: Sample Apps/Storefront/Storefront/TotalsViewController.swift [hunk 2]
```diff
@@ -50,4 +61,53 @@ class TotalsViewController: UIViewController {
...
+    // ----------------------------------
+    //  MARK: - Actions -
+    //
+    @IBAction func applePayAction(_ sender: Any) {
+        self.delegate?.totalsController(self, didRequestPaymentWith: .applePay)
+    }
 }
+
+
+//private func fetchCurrencyDescription(completion: @escaping (CurrencyDescription?) -> Void) {
```
Comment (davidmuzi) [inline Sample Apps/Storefront/Storefront/TotalsViewController.swift]: 🔥

File: Sample Apps/Storefront/Storefront/Graph.swift [hunk 3]
```diff
@@ -92,4 +95,25 @@ final class Graph {
...
         return task
     }
+    
+    // ----------------------------------
+    //  MARK: - Checkout -
+    //
+    @discardableResult
+    func createCheckout(with cartItems: [CartItem], completion: @escaping (Storefront.Checkout?) -> Void) -> URLSessionDataTask {
+        let mutation = GraphQuery.mutationForCreateCheckout(with: cartItems)
+        let task     = self.client.mutateGraphWith(mutation) { response, erro in
```
Comment (davidmuzi) [inline Sample Apps/Storefront/Storefront/Graph.swift]: "error"

File: Sample Apps/Storefront/Storefront/CartViewController.swift (hunk 1 above)
Comment (dbart01) [inline Sample Apps/Storefront/Storefront/CartViewController.swift]: Yeah I was thinking a `Info.plist` entry but that will be addressed in future PRs.

**Pull request merged**